cli/CampaignConfigValidation/ValidateCampaignConfigCommand.php

```diff
@@ -0,0 +1,86 @@
+<?php
+
+declare( strict_types = 1 );
+
+namespace WMDE\Fundraising\Frontend\Cli\CampaignConfigValidation;
+
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Input\InputArgument;
+use Symfony\Component\Console\Input\InputDefinition;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+use Symfony\Component\Filesystem\Exception\FileNotFoundException;
+use WMDE\Fundraising\Frontend\Factories\FunFunFactory;
+use WMDE\Fundraising\Frontend\Infrastructure\ConfigReader;
+
+/**
+ * @license GNU GPL v2+
+ */
+class ValidateCampaignConfigCommand extends Command {
+
+	const NAME = 'validate-campaign-config';
+	const DEFAULT_SCHEMA = __DIR__ . '/../../app/config/schema.json';
+	const ERROR_RETURN_CODE = 1;
+	const OK_RETURN_CODE = 0;
+
+	protected function configure(): void {
+		$this->setName( self::NAME )
+			->setDescription( 'Validate configuration files' )
+			->setHelp(
+				'This command merges the specified configuration files and validates them against a JSON schema'
+			)
+			->setDefinition(
+				new InputDefinition(
+					[
+						new InputArgument( 'environment', InputArgument::OPTIONAL, '', 'prod' )
+					]
+				)
+			);
+	}
+
+	protected function execute( InputInterface $input, OutputInterface $output ): int {
+		$environment = $input->getArgument( 'environment' );
+		$factory = $this->getFactory( $environment );
+		try {
```

Comment:
You might want to move this try...catch block into a `LoggingCampaignConfigurationLoader` that implements the `CampaignConfigurationLoader` interface and is a wrapper around `CampaignConfigurationLoader`. Its `loadCampaignConfiguration` could catch exactly the exceptions `CampaignConfigurationLoader` throws and translate them into error messages (and letting other, unforeseen exceptions bubble up). That way, you can even unit test the error handling by passing in a dummy `ThrowingCampaignConfigurationLoader` in the test.

The Campaign Loader can be set in the factory before calling `$factory->getCampaignCollection();`